src/contradictory_claims/claims/extract_claims.py

```diff
@@ -11,10 +11,13 @@
 from nltk import sent_tokenize
 from torch.nn import Linear
 
-from .predictors import ClaimCrfPredictor
 from .dataset_reader import CrfPubmedRCTReader
+from .models import DiscourseCrfClassifier  # noqa: F401  # this is the model, which would be loaded
+from .predictors import ClaimCrfPredictor
 from .utils import MODEL_PATH, WEIGHT_PATH
 
+nltk.download('punkt')  # creating issue in travis
```

Comment:
travis CI was giving issue with language model not found, so downloading here only